docs/docs/platform/step-filter.md

```diff
@@ -0,0 +1,137 @@
+  to send notifications only to online subscribers.
+
+### Steps to set filter
+
+1. Create a step in your notification workflow. This step will be the location where
+   you will apply your filter.
+
+2. Add a filter to the step. This will allow you to specify the criteria that must be
+   met for the notification to be delivered.
+
+   <div align="center">
+     <a href="https://novu.co" target="_blank">
+     <picture>
+       <source media="(prefers-color-scheme: dark)" srcset="/img/platform/filter/dark-filter-section.png"/>
+       <img src="/img/platform/filter/light-filter-section.png" width="1280" alt="Logo"/>
+     </picture>
+     </a>
+   </div>
+
+3. If you want to add multiple filters, you will need to choose a logical condition
+   of AND or OR (1). This will determine how the filters are applied to the notification.
+   For example, if you choose AND, the notification will only be delivered if all of the
+   filters are met. If you choose OR, the notification will be delivered if any of the
+   filters are met.
+
+4. Add a rule to the filter (2). This will specify the criteria that must be met for the
+   notification to be delivered. You can use the 'payload', 'subscriber', or 'webhook' field
+   as the basis for your rule. For example, you might specify that the 'payload' field must
+   contain a certain keyword or that the 'subscriber' must be in a specific group.
+
+> If only one rule is provided, a logical condition of AND or OR is not mandatory.
+
+<div align="center">
+  <a href="https://novu.co" target="_blank">
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="/img/platform/filter/dark-add-filter-section.png"/>
+    <img src="/img/platform/filter/light-add-filter-section.png" width="680" alt="Logo"/>
+  </picture>
+  </a>
+</div>
+
+A rule is composed of the following elements:
+
+- On field: This specifies which field the rule should be applied to, such as 'payload',
+  'subscriber', or 'webhook' response.
+
+- Key field: This specifies the object key that the rule should be applied to. For example,
+  if the 'On' field is 'subscriber', the 'Key' field might be 'name', 'email', etc.
+
+- Operator field: This specifies the comparison operator to use when evaluating the rule.
+  Options include 'Equal', 'Not equal', 'Larger', 'Smaller', 'Larger or equal', 'Smaller or equal',
+  'Contains', and 'Not contains'.
+
+- Value field: This specifies the value to compare to the object
+  key value using the selected operator.
+
+Example: In this filter rule, the notification will only be delivered if the
+webhook response includes a field called 'isOnline' that is set to 'true'.
+
+<div align="center">
+  <a href="https://novu.co" target="_blank">
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="/img/platform/filter/dark-webhook-rule-example.png"/>
+    <img src="/img/platform/filter/light-webhook-rule-example.png" width="680" alt="Logo"/>
+  </picture>
+  </a>
+</div>
```

Comment:
```suggestion
```html
<div align="center">
  <a href="https://novu.co" target="_blank">
  <picture>
    <source media="(prefers-color-scheme: dark)" srcset="/img/platform/filter/dark-webhook-rule-example.png"/>
    <img src="/img/platform/filter/light-webhook-rule-example.png" width="680" alt="Logo"/>
  </picture>
  </a>
</div>
```
```
Basically add the code template markdown.

Reply:
Same here